datasets/tatoeba/README.md

```diff
@@ -0,0 +1,505 @@
+-max
+-mdf
+-mfe
+-mg
+-mgm
+-mh
+-mhr
+-mi
+-mic
+-min
+-mk
+-ml
+-mn
+-mni
+-mnw
+-moh
+-mr
+-mt
+-mvv
+-mwl
+-mww
+-my
+-myv
+-na
+-nah
+-nan
+-nb
+-nch
+-nds
+-ngt
+-ngu
+-niu
+-nl
+-nlv
+-nn
+-nog
+-non
+-nov
+-npi
+-nst
+-nus
+-nv
+-ny
+-nys
+-oar
+-oc
+-ofs
+-ood
+-or
+-orv
+-os
+-osp
+-ota
+-otk
+-pa
+-pag
+-pal
+-pam
+-pap
+-pau
+-pcd
+-pdc
+-pes
+-phn
+-pi
+-pl
+-pms
+-pnb
+-ppl
+-prg
+-ps
+-pt
+-qu
+-quc
+-qya
+-rap
+-rif
+-rm
+-rn
+-ro
+-rom
+-ru
+-rue
+-rw
+-sa
+-sah
+-sc
+-scn
+-sco
+-sd
+-sdh
+-se
+-sg
+-sgs
+-shs
+-shy
+-si
+-sjn
+-sl
+-sm
+-sma
+-sn
+-so
+-sq
+-sr
+-stq
+-su
+-sux
+-sv
+-swg
+-swh
+-syc
+-ta
+-te
+-tet
+-tg
+-th
+-thv
+-ti
+-tig
+-tk
+-tl
+-tlh
+-tly
+-tmr
+-tmw
+-tn
+-to
+-toi
+-toki
+-tpi
+-tpw
+-tr
+-ts
+-tt
+-tts
+-tvl
+-ty
+-tyv
+-tzl
+-udm
+-ug
+-uk
+-umb
+-ur
+-uz
+-vec
+-vep
+-vi
+-vo
+-vro
+-wa
+-war
+-wo
+-wuu
+-xal
+-xh
+-xqa
+-yi
+-yo
+-yue
+-zlm
+-zsm
+-zu
+-zza
+licenses:
+- cc-by-2.0
+multilinguality:
+- multilingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- machine-translation
+---
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** http://opus.nlpl.eu/Tatoeba.php
+- **Repository:** None
+- **Paper:** http://www.lrec-conf.org/proceedings/lrec2012/pdf/463_Paper.pdf
+- **Leaderboard:** [More Information Needed]
+- **Point of Contact:** [More Information Needed]
+
+### Dataset Summary
+
+
+To load a language pair which isn't part of the config, all you need to do is specify the language code as pairs.
+You can find the valid pairs in Homepage section of Dataset Description: http://opus.nlpl.eu/Tatoeba.php
+E.g.
+
+`dataset = load_dataset("tatoeba", lang1="en", lang2="he")`
+
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+Here are some examples of questions and facts:
+
```

Comment:
```suggestion
[More Information Needed]
```

I might have missed some of these issues, could you take a look at the PR we merged ?